frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -97,13 +113,28 @@ export class Wizard {
   }
 
   async selectWorkloadProfile(workloadProfile: string) {
-    await selectOptionByText(view.workloadProfileSelect, workloadProfile);
+    await selectItemFromDropdown(view.workloadProfileSelect, view.dropDownItem(workloadProfile));
+  }
+
+  async disableGoldenImageCloneCheckbox() {
+    if (
+      (await view.goldenImageCloneCheckbox.isPresent()) &&
```

Comment:
this is called right after the os selection which might make this appear. IMO we should wait at least 1sec if it does not appear